Src/AutoFixture.NUnit3/DefaultTestCaseParameterBuilder.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using NUnit.Framework.Interfaces;
+using NUnit.Framework.Internal;
+
+namespace Ploeh.AutoFixture.NUnit3
+{
+    /// <summary>
+    /// Default builder used by <see cref="AutoDataAttribute"/> and <see cref="InlineAutoDataAttribute"/> to create
```

Comment:
Please rewrite doc to not mention who uses this type and how. Instead simply document the behavior of this strategy as if it would be one among many other similar strategies.